### DISCUSSION TITLE
dagre布局同级combo内部节点如果有连接, 布局会乱, 如何解决? 先感谢各位了

### DISCUSSION THREAD

**Discussion by CorneliusYang** [antvis/G6#5198] [General]
![pic](https://github.com/antvis/G6/assets/83103653/ebb1b0f9-4683-40c4-9ff6-5c4f1d38c71d)
目前使用的是comboCombined布局, 内外布局都是dagre, 如上图, 如果combo2的赵六1节点跟combo3的里斯111连线的话, 布局会直接打乱,下面这个图是连接之后的样子, 有没有什么办法让他们能只连线不打乱布局, 先感谢各位大佬了.
![pic2](https://github.com/antvis/G6/assets/83103653/d0029b05-20d1-45be-8c85-0eebb384c415)

**Comment by Aarebecca** [comment]
重复的问题： https://github.com/antvis/G6/discussions/5197
